README.md

```diff
@@ -1,30 +1,58 @@
-<!--
-  -- Copyright 2020 Contributors to the Parsec project. 
-  -- SPDX-License-Identifier: Apache-2.0
---->
 # Parsec Tool
 
 This repository contains a tool to communicate with the [Parsec
 service](https://github.com/parallaxsecond/parsec) on the command-line.
 
-For demos and to test the Parsec service, you might want to change the Parsec endpoint location.
-For that, set the `PARSEC_SERVICE_ENDPOINT` environment variable to correction endpoint.
+## Getting started
+
+To compile and list the available commands:
+
+```
+$ cargo build
+$ ./target/debug/parsec-tool --help
+```
```

Comment:
Maybe you can just show the instruction for installing it? `cargo install --path ./` or smth like that. The `./target/debug...` thing looks kinda unfinished

Reply:
I can replace with `cargo run`? I am a bit afraid of describing an installation method for something as early as `parsec-tool` 😨

Reply:
Sure, I don't mind - in that case just leave it as is